graph/db/graph_cache_test.go

```diff
@@ -134,7 +135,7 @@ func assertCachedPolicyEqual(t *testing.T, original *models.ChannelEdgePolicy,
 		t, original.FeeProportionalMillionths,
 		cached.FeeProportionalMillionths,
 	)
-	require.Equal(
-		t, route.Vertex(original.ToNode), cached.ToNodePubKey(),
-	)
+	if original.ToNodePubKey != nil {
```

Comment:
Nit: Should we allow nil here in the first place ?

`require.NotNil` or something ?